### DISCUSSION TITLE
How to detect for specific reactions?

### DISCUSSION THREAD

**Discussion by jellytoaster** [Rapptz/discord.py#6330] [Q&A]
I'm making a discord bot, and i'm making a setup command to set up roles and channels. I want them to confirm they are doing that by doing a reaction check (check for reactions). I want it so it can check for reacting the checkmark reaction and the cross reaction so they can choose if they want to set the roles and channels up. Unfortunately, MY CODE IS NOT WORKING!!!! help

**Comment by Rapptz** [comment]
Generally speaking before asking for help here it's recommended to show code on what you've tried. Otherwise this would spiral into just asking people to write code for you which is not a good use of maintainer time. 

Note that the example for [`Client.wait_for`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Client.wait_for) showcases how to do this with different reactions, meaning with a bit of messing around it should be possible to solve this issue too.

**Reply by jellytoaster** [reply]
Never mind, i used a client var instead of bot var :|
